int64 hri_status
int64 audio_message
int64 safety_action
int64 human_command
int64 critical_index
int64 goal_index

